src/php7/solr_functions_helpers.c

```diff
@@ -18,7 +18,9 @@
 
 #include "php_solr.h"
 
+#ifndef PHP_WIN32
 ZEND_EXTERN_MODULE_GLOBALS(json)
+#endif
```

Comment:
Is this needed on any platform?

Reply:
I really would not know, so I only removed it for Windows.

Reply:
It was added 11 years ago in https://github.com/php/pecl-search_engine-solr/commit/7a9fa5104e1265a10983bd902c111e0010d21857#diff-270247253318bab6ceee9c42f645c7a50013349bc086b275a5b3a384be69f42eR23
Maybe it was ever needed when JSON was loaded dynamically.

Reply:
Doesn't look like that declaration is ever used (or has been when it has been added). It doesn't really hurt to keep it, though.

Reply:
It was causing a compiler warning, as you can also see in the logs of building the extension for PHP 7.4:

> ext\solr\src\php7\solr_functions_helpers.c(21): warning C4273: 'json_globals': inconsistent dll linkage
> C:\php-snap-build\php-src\php74\x64\php-src\ext/json/php_json.h(94): note: see previous definition of 'json_globals' (compiling source file ext\solr\src\php7\solr_functions_helpers.c)

https://windows.php.net/downloads/pecl/releases/solr/2.5.1/logs/
That is why I removed it for Windows